src/controller/org.controller/org.controller.js

```diff
@@ -81,6 +81,53 @@ async function getOrg (req, res, next) {
   }
 }
 
+async function getUsers (req, res, next) {
+  try {
+    const shortName = req.ctx.org
+    const orgShortName = req.ctx.params.shortname
+    const orgRepo = req.ctx.repositories.getOrgRepository()
+    const userRepo = req.ctx.repositories.getUserRepository()
+    const orgUUID = await orgRepo.getOrgUUID(orgShortName)
+    let returned = false
+
+    if (!orgUUID) {
+      returned = true
+      logger.info(orgShortName + ' organization does not exist.')
```

Comment:
This doesn't need to be resolved in this pull request, but, why is this `logger` call just a string, while the one below is in the format `{ message: string}`?